canonical-web-and-design/canonical.com
# 86 update careers techops

## Done

- Update `/careers/tech-ops` page
- Update `/careers/sales` page
- Update all other careers pages with copy doc link and metadata if available
- Remove some unnecessary markup

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/careers/tech-ops and http://0.0.0.0:8002/careers/sales
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [tech-ops copy](https://docs.google.com/document/d/1wzuu4pnd2Sx2BDONaaR9EaYUSjx7diTu6PzvYYwbMA8/edit) and [sales copy](https://docs.google.com/document/d/1fGBFiIfy7C58UXcJ0Q7q6VKxClxBfPyeWBSyO9GrStI/edit)


## Issue / Card

Fixes #86 
Fixes #105
